packages/teams-js/src/public/interfaces.ts

```diff
@@ -616,7 +616,7 @@ export interface Context {
    * @deprecated
    * As of 2.0.0, please use {@link app.AppInfo.sessionId | app.Context.app.sessionId} instead
    *
-   * Unique ID for the current session for use in correlating telemetry data.
+   * Unique ID for the current session for use in correlating telemetry data. A session corresponds to Teams app life cycle and gets created afresh when a new webview/iframe is created and ends upon the destruction of the said webview/iframe in which the app lives.
```

Comment:
> A session corresponds to Teams app life cycle and gets created afresh when a new webview/iframe is created and ends upon the destruction of the said webview/iframe in which the app lives. [](http://example.com/codeflow?start=77&length=188)

@erikadoyle : any suggestions on this updated comment? What info do you think would be most useful to developers to know here?

Reply:
How about: 

A session corresponds to the lifecycle of an app. A new session begins upon the creation of a webview (on Teams desktop) or iframe (in Teams mobile) hosting the app, and ends when it is destroyed.

@AE-MS, @sakov-ms I'm curious--when is the webview/iframe torn down? When user navigates away from the app within Teams? When user restarts Teams?